src/main/flight/imu.c

```diff
@@ -749,6 +749,13 @@ void imuUpdateAttitude(timeUs_t currentTimeUs)
 }
 #endif // USE_ACC
 
+// Angle in between nose axis of the craft and the horizontal plane in ground reference.
+// Poitive angle - nose down, negative angle - nose up.
+float getSinTiltAngle(void)
```

Comment:
`getCosTiltAngle` gets angle between Body Z and World up - both roll and pitch cause tilt. `getSinNoseUpAngle`? But that sounds terrible

Reply:
oh yeah, they are different anles... How about "getSinDiveAngle"? Diving down - positive dive angle. Negative dive angle - not diving.

Reply:
@limon aren't you mostly concerned about pitch down?  If the wing was side-slipping, like roll down but (say) with no pitch down, in that case you'd not have much authority?  I am a little unsure about giving equal weight to the roll element in the tilt factor.

Reply:
@ctzsnooze this was my bad to name `getSinTiltAngle`, because it actually returns the "dive" angle, basically how far the nose it down. Roll can be a factor, but not by itself. For example if you roll 90 degrees, then yaw will affect how far nose is down.

Reply:
Thanks for the clarification.
Maybe 'getSinNoseDownAngle` or just `getNoseDownAngle` might be OK :-)

Reply:
Agree function name should not include technical implementation detail

Reply:
i kinda like that "Dive angle" idea...
"getSinDiveAngle" sounds better than "getSinNoseDown"?